ncm-afsclt/src/main/perl/afsclt.pm

```diff
@@ -85,14 +88,27 @@ sub Configure_Cache {
     my $new_cache       = 0;     # in 1k blocks.
     my $file_afsmount   = '';
 
-    if ( $config->elementExists("$PREFIX/cachesize") ) {
-        $new_cache = $config->getValue("$PREFIX/cachesize");    #new cache size
-    }
-    else {
-        $self->info("Cannot get CDB $PREFIX/cachesize - not setting cache size");
+    my $afs_config = $config->getElement($PREFIX)->getTree();
```

Comment:
I'm working on it, not sure if it will make it in 16.6... I first want to add unit test before making the changes everywhere in the code.